client/src/config/wagtailConfig.ts

```diff
@@ -28,6 +19,19 @@ function getWagtailConfig() {
     // valid JSON, ignore it and return an empty object.
     return {};
   }
-}
+};
 
-export const WAGTAIL_CONFIG = getWagtailConfig();
+export const WAGTAIL_CONFIG = getWagtailConfig() as WagtailConfig;
+
+export const { ADMIN_API } = WAGTAIL_CONFIG;
+export const { ADMIN_URLS } = WAGTAIL_CONFIG;
+
+// Maximum number of pages to load inside the explorer menu.
+export const MAX_EXPLORER_PAGES = 200;
+
+export const LOCALE_NAMES = new Map();
+
+/* eslint-disable-next-line camelcase */
+WAGTAIL_CONFIG.LOCALES?.forEach(({ code, display_name }) => {
+  LOCALE_NAMES.set(code, display_name);
+});
```

Comment:
We can have a smaller diff if these constants are left at the top

Reply:
I cannot really do this without refactoring the usage of the function and then adding an eslint ignore for 'use before declare'.

Also, there are still a few lines that will have significant diffs as we are are not destructuring from the global it does not really make a the diff much cleaner.

I have also reviewed this code again and done a refinement on the exports and will force push. I have split out the 'rename' to its own commit so the file history will be easier to track through the changes.